persia/backend.py

```diff
@@ -64,6 +64,16 @@ def set_configuration(
     def register_optimizer(self, optimizer: PyOptimizerBase):
         self.nats_publisher.register_optimizer(optimizer)
 
+    def dump_embedding(self, dst_dir: str, blocking: bool = False):
+        self.rpc_client.dump_embedding(dst_dir)
+        if blocking:
+            self.rpc_client.wait_for_dump_embedding()
+
+    def load_embedding(self, dst_dir: str, blocking: bool = True):
+        self.rpc_client.load_embedding(dst_dir)
+        if blocking:
+            self.rpc_client.wait_for_load_embedding()
```

Comment:
Sure it is a good point. However, I don't know that do people really like to write asynchronous code when they train a model. Would it be better if Persia handle asynchronous event rather than users?

Reply:
That just provide `Backend.wait_for_load_embedding` function now

Reply:
Solved in https://github.com/PersiaML/PersiaML/pull/18/commits/977a69174f1daa6e96ddf5f90a62fe00d1c11e0a